packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -1,252 +1,194 @@
+  border-radius: $choice-border-radius;
+  cursor: pointer;
+  display: grid;
+  height: $choice-size;
+  justify-items: center;
+  margin: 0;
+  place-content: center;
+  position: relative;
+  width: $choice-size;
+
+  &::before {
+    box-shadow: inset 0 0 1em 1em $choice-checked-background-color;
+    content: '';
+    grid-area: 1/-1;
+    height: $choice-size;
+    opacity: 0;
+    transition: 120ms opacity ease-in-out;
+    width: $choice-size;
   }
 
-  // Errored choice
+  // Error
   @if $ds-include-choice-error-highlight {
-    &.ds-c-choice--error + label::before {
+    &.ds-c-choice--error {
       border-color: $color-error;
     }
   }
 
-  // Checked choice
-  &:checked + label {
+  // Checked
+  &:checked {
     &::before {
-      background-color: $choice-checked-background-color;
-      border-color: $choice-checked-border-color;
-
-      /* stylelint-disable */
-      @media (-ms-high-contrast: active), (forced-colors: active) {
-        border-color: ButtonText;
-      }
-      /* stylelint-enable */
+      opacity: 1;
     }
 
-    &::after {
-      opacity: 1;
+    &:focus {
+      border-color: $color-primary-darker;
     }
   }
 
-  // Disabled choice
-  &:disabled + label {
+  // Disabled
+  &:disabled {
+    background-color: $choice-disabled-background-color;
+    border-color: $choice-disabled-border-color;
     color: $color-muted;
     cursor: not-allowed;
-
-    &::before {
-      background-color: $choice-disabled-background-color;
-      border-color: $choice-disabled-border-color;
-    }
-
-    /* stylelint-disable */
-    @media (-ms-high-contrast: active), (forced-colors: active) {
-      color: GrayText;
-
-      &::before {
-        background-color: Window;
-        border-color: GrayText;
-      }
-    }
-    /* stylelint-enable */
   }
 
-  // Focused choice
-  &:focus + label::before {
+  // Focus
+  &:focus {
     @include focus-styles;
   }
 }
 
-.ds-c-choice--inverse {
-  // Choice checkbox/radio button
-  + label::before {
-    background-color: $choice-background-color-inverse;
-    border-color: $choice-border-color-inverse;
+// Checkbox styles
+.ds-c-choice[type='checkbox'] {
+  &::after {
+    background-color: transparent;
+    border: solid;
+    border-color: $color-white;
+    border-top-color: transparent;
+    border-width: 0 0 4px 4px;
+    content: '';
+    grid-area: 1/-1;
+    height: 4px;
+    opacity: 0;
+    transform: rotate(-45deg) translateX(0.075em);
+    width: 10px;
   }
 
-  // Errored inverse choice
-  @if $ds-include-choice-error-highlight {
-    &.ds-c-choice--error + label::before {
-      border-color: $color-error-light;
+  &:checked {
+    &::after {
+      opacity: 1;
     }
+
+    /* stylelint-disable max-nesting-depth, selector-max-specificity */
+    &:focus::before {
+      height: $choice-size - ($choice-border-width * 2);
+      width: $choice-size - ($choice-border-width * 2);
+    }
+    /* stylelint-enable max-nesting-depth, selector-max-specificity */
```

Comment:
Can remove all stylelint disables like this, and merge master which includes the eslint/stylelint updates.

Reply:
Merging master results in an error:
> Error: Could not find "stylelint-config-recommended". Do you need a `configBasedir`?

Is there a step I'm missing to get master updated?

Reply:
You'll have to do a yarn install after merging, should fix it.